pkg/backoff/backoff.go

```diff
@@ -114,6 +119,16 @@ func ClusterSizeDependantInterval(baseInterval time.Duration, numNodes int) time
 
 // Wait waits for the required time using an exponential backoff
 func (b *Exponential) Wait(ctx context.Context) error {
+	resetDuration := b.BackoffResetDuration
+	if resetDuration == time.Duration(0) {
+		resetDuration = 10 * time.Minute
```

Comment:
> One option would be to set this to `2 * b.Max`, but I think it might be better to not assume a default value and have users set `BackoffResetDuration` explicitly.

I think only do reset when BackoffResetDuration is set and if BackoffResetDuration less than force set BackoffResetDuration to 2 * b.Max

Reply:
Sounds good to me too, as long as the behavior is documented.

Reply:
Yeah, let's only do the reset if the value is set. Let's also just ignore it if it is less than Max, since that is a user error. I added some sample documentation in the comment above.